File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/createnew/fragments/createnew/state/CreateNewState.java
package ua.meugen.android.pdfcomposer.ui.activities.createnew.fragments.createnew.state;

import java.util.List;

import ua.meugen.android.pdfcomposer.model.data.PageContent;
import ua.meugen.android.pdfcomposer.ui.activities.base.fragment.state.MvpState;


public interface CreateNewState extends MvpState {

    List<PageContent> getPages();

    void setPages(List<PageContent> pages);
}

File: app/src/main/java/ua/meugen/android/pdfcomposer/ui/activities/createnew/fragments/createnew/state/CreateNewState.kt
package ua.meugen.android.pdfcomposer.ui.activities.createnew.fragments.createnew.state

import ua.meugen.android.pdfcomposer.model.data.PageContent
import ua.meugen.android.pdfcomposer.ui.activities.base.fragment.state.MvpState


interface CreateNewState : MvpState {

    var pages: List<PageContent>?
}
